Decred Wallet/view_util/DropMenuButton.swift

```diff
@@ -151,16 +157,17 @@ class DropMenuButton: UIButton, UITableViewDelegate, UITableViewDataSource
     
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell
     {
-        let itemLabel = UILabel(frame: CGRect(x: 10, y: 0, width: frame.width - 10, height: frame.height))
+        let itemLabel = UILabel(frame: CGRect(x: 10, y: 0, width: 300, height: frame.height))
         itemLabel.textAlignment = NSTextAlignment.left
         itemLabel.text = items[(indexPath as NSIndexPath).row]
         itemLabel.font = UIFont(name: "SourceSansPro-Regular", size: 16)
         itemLabel.textColor = UIColor.black
         
+        self.minTableWidth = max(self.minTableWidth, itemLabel.intrinsicContentSize.width + 20)
```

Comment:
Is changing this value here guaranteed to trigger relayout? If not, add a `didSet` callback to `minTableWidth` to call `self.fixLayout()` whenever the `minTableWidth` value changes.